disnake/widget.py

```diff
@@ -132,21 +133,21 @@ class WidgetMember(BaseUser):
     Attributes
     ----------
     id: :class:`int`
-        The member's ID.
+        The member's anonymized ID.
     name: :class:`str`
         The member's username.
     discriminator: :class:`str`
-        The member's discriminator.
-    bot: :class:`bool`
-        Whether the member is a bot.
+        The member's anonymized discriminator.
     status: :class:`Status`
         The member's status.
     nick: Optional[:class:`str`]
         The member's nickname.
-    avatar: Optional[:class:`str`]
-        The member's avatar hash.
+    avatar: Optional[:class:`Asset`]
+        The user's avatar. The size can be chosen using :func:`Asset.with_size`,
+        however the format is always static and cannot be changed
+        through :func:`Asset.with_format` or similar methods.
     activity: Optional[Union[:class:`BaseActivity`, :class:`Spotify`]]
```

Comment:
I'm not sure the api will provide spotify statuses, but I suppose we shouldn't take that risk if the code can return a spotify instance.

Reply:
I haven't seen any widget members with an activity that has more fields than just `{"name": "Celeste"}`, but for now I didn't want to touch that part of the code.
It would be a breaking change now to e.g. use `Optional[str]` instead, and would be a breaking change *again* if they decide to provide full activity objects in the future.
The client also only processes the `name` field in `game`/`activity`, at least currently that seems to be all there is to it.

Reply:
Makes sense.